src/components/pages/RockectDisplay.css

```diff
@@ -0,0 +1,47 @@
+.main {
+  display: flex;
+  gap: 2vw;
+  margin: 2vw;
+}
+
+.container {
+  border-top: solid 1px lightgray;
+}
+
+.main img {
+  height: 14vw;
+  width: 15vw;
+}
+
+.rocketDescription {
+  text-align: left;
+  line-height: 1;
+  margin-left: 1rem;
+}
+
+/* .main button {
+  background-color: rgb(17, 116, 255);
+  padding: 0.5rem 1rem;
+  outline: none;
+  border: none;
+  color: #fff;
+  border-radius: 0.3rem;
+} */
```

Comment:
- [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code 😉

Reply:
Noted

Reply:
Solved